src/pages/PrivacyPolicyPage.js

```diff
@@ -0,0 +1,7 @@
+import PrivacyPolicyComponent from "../components/PrivacyPolicy/PrivacyPolicy";
+import React from "react";
+
+const PrivacyPolicyPage = () => (
+    <PrivacyPolicyComponent />
```

Comment:
No need to separate into `*Page` and `*Component`, as this is a pretty simple page/component, just put it directly here